src/trig.c

```diff
@@ -1,9 +1,6 @@
 #include "global.h"
 #include "trig.h"
 
-// Converts a number to Q8.8 fixed-point format
```

Comment:
Why move one and not the other?

Reply:
So far I didn't encounter any other situation that would require this macro.